<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>401</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Serial Disposer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>441</width>
      <height>381</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QComboBox" name="comboBox_ports"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_refresh">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Refresh</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_baudrate">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Select baudrate:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_baudrate">
         <property name="currentText">
          <string notr="true">9600</string>
         </property>
         <property name="currentIndex">
          <number>4</number>
         </property>
         <item>
          <property name="text">
           <string>300</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1200</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2400</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>4800</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>9600</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>19200</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>38400</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>57600</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>74880</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>115200</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>230400</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>250000</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>500000</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1000000</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2000000</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_connect">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Connect</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTextEdit" name="textEdit_terminal"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLineEdit" name="lineEdit_send"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_send">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Send</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
